lib/surface/compiler/tokenizer.ex

```diff
@@ -530,9 +530,12 @@ defmodule Surface.Compiler.Tokenizer do
   end
 
   ## handle_attr_name
+  defp handle_attr_name(<<"/", _rest::binary>>, _column, []) do
+    {:error, "unexpected closing tag delimiter `/`"}
+  end
 
-  defp handle_attr_name(<<c::utf8, _rest::binary>>, _column, []) when c in @name_stop_chars do
-    {:error, "expected attribute name"}
+  defp handle_attr_name(<<c::utf8, _rest::binary>> = text, _column, []) when c in @name_stop_chars do
+    {:error, "expected attribute name, got: `#{text}`"}
```

Comment:
I wonder if  it wouldn't be better to use just:

```
{:error, "expected attribute name, got: `#{c}`"}
```

After all `text` will be the rest of the whole template which might pollute the error message a lot.

Reply:
We definitely could, but it would always be a single character, in this case "=". I'm happy to change to that.

Btw I'll also need to convert from the ascii code to a string

Reply:
walked it back to your suggestion @msaraiva